### PR TITLE
SAM-3350 Timed assessment progress bar includes hard-coded label "Time's Up"

### PR DESCRIPTION
**Pull request by joaquinmarques**

### CODE REVIEW

**Comment by ottenhoff** [comment]
This looks fine. Can you fix tabs/spaces issues?

**Comment by joaquinmarques** [comment]
Updated

File: samigo/samigo-app/src/webapp/WEB-INF/samigo.tld [hunk 1]
```diff
@@ -140,6 +140,11 @@
     <required>true</required>
     <rtexprvalue>false</rtexprvalue>
   </attribute>
+  <attribute>
+    <name>timeUpMessage</name>
+    <required>true</required>
+    <rtexprvalue>false</rtexprvalue>
+  </attribute>
```
Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/WEB-INF/samigo.tld]: does this really require a change to samigo.tld?!

Comment (joaquinmarques) [reply samigo/samigo-app/src/webapp/WEB-INF/samigo.tld]: Yes because it is necessary specify the timeUpMessage attribute for samigo:timerBar

**Pull request merged**